rst_files/orth_proj.rst

```diff
@@ -70,7 +70,7 @@ For a linear subspace  :math:`S \subset \mathbb{R}^n`, we call :math:`x \in \mat
 .. figure:: /_static/figures/orth_proj_def2.png
     :scale: 50%
 
-The **orthogonal complement** of linear subspace :math:`S \subset RR^n` is the set :math:`S^{\perp} := \{x \in \mathbb{R}^n \,:\, x \perp S\}`
+The **orthogonal complement** of linear subspace :math:`S \subset R^n` is the set :math:`S^{\perp} := \{x \in \mathbb{R}^n \,:\, x \perp S\}`
```

Comment:
Good catch. Although I think should be changed to `\RR^n`? Whatever the rest of them use in this lecture.

Reply:
Thanks. Apparently \RR^n isn't work properly. I will change them to \mathbb{R}^n like above. Is it ok?

Reply:
Yep, that's perfect. A while back we eliminated the shortcodes (`\RR`, `\EE`, etc.) to just use the raw LaTeX.